packages/venia-ui/lib/components/AccountInformationPage/accountInformationPage.css

```diff
@@ -0,0 +1,45 @@
+.root {
+    display: grid;
+    padding: 2.5rem 3rem;
+    row-gap: 2rem;
+}
+
+.title {
+    justify-self: center;
+    font-family: var(--venia-global-fontFamily-serif);
+    font-weight: var(--venia-global-fontWeight-bold);
+}
+
+.lineItems {
+    display: grid;
+    grid-gap: 0.75rem;
+    grid-template-columns: 1fr auto;
+    line-height: 1.5rem;
+}
+
+.lineItemLabel {
+    font-weight: var(--venia-global-fontWeight-bold);
+}
+
+.editInformationButton {
+    composes: root_normalPriority from '../Button/button.css';
+    margin: auto;
+}
+
+@media (min-width: 961px) {
+    .cardArea {
+        display: grid;
+        gap: 1rem;
+        grid-template-columns: 1fr 1fr 1fr;
```

Comment:
What is the reason for the three columns, or the grid view, at all? 

[![Image from Gyazo](https://i.gyazo.com/159ae8499bb00a92bf298d3d8ffa394b.png)](https://gyazo.com/159ae8499bb00a92bf298d3d8ffa394b)

Reply:
what is the best css we should use here?

Reply:
I think it actually might just be the simplest way to get the style we're looking for on this page. It was signed off so I'll resolve this comment.

Reply:
Thanks you